### PR TITLE
Moments cntrlr & method

### PR DESCRIPTION
**Pull request by Stef-Github-Acc**

### CODE REVIEW

File: app/controllers/moments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class MomentsController < ApplicationController
+  def new
+    @moment = Moment.new
+  end
+
+  def create
+    @moment = Moment.new(params[:moment])
```
Comment (ddlire) [inline app/controllers/moments_controller.rb:7]: moment_params

File: app/controllers/moments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    @moment.save
```
Comment (ddlire) [inline app/controllers/moments_controller.rb:8]: redirect to?

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
```
Comment (ddlire) [inline app/controllers/users_controller.rb:3]: @trips = @user.trips

File: app/controllers/moments_controller.rb (hunk 1 above)
Comment (Stef-Github-Acc) [reply app/controllers/moments_controller.rb:7]: Updated.

File: app/controllers/moments_controller.rb (hunk 2 above)
Comment (Stef-Github-Acc) [reply app/controllers/moments_controller.rb:8]: Updated

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (Stef-Github-Acc) [reply app/controllers/users_controller.rb:3]: Updated.

**Pull request merged**